include/uitsl/mpi/audited_routines.hpp

```diff
@@ -55,12 +55,12 @@ inline void UITSL_Add_error_string(Args&&... args) {
 
 /// @private
 /// Wraps corresponding MPI* call with debug-mode error code check.
-template<typename... Args>
-inline void UITSL_Address(Args&&... args) {
-  uitsl::mpi_audit(
-    MPI_Address(std::forward<Args>(args)...)
-  );
-}
+// template<typename... Args>
```

Comment:
uncomment --- I think this is only broken in your environment (for mysterious reasons)